### ISSUE TITLE
Add a "Report issue" button

### ISSUE THREAD

**Issue by madoar** [PhoenicisOrg/phoenicis#1776]
Currently there is no easy way for a user to report an issue (either with Phoenicis directly or with a script). Therefore I suggest to add a button to the `ApplicationDetailsPanel` through which the user can report an issue with a script.

Open questions:
- how would the button know the script which lead to the bug in case the application has multiple scripts?
- can we somehow fetch/add the corresponding logs?

**Comment by plata** [comment]
The button does not know anything. It's just a link to https://github.com/PhoenicisOrg/scripts/issues/new.

**Comment by Kreyren** [comment]
It should report what WINE version is used, list of dependencies, GPU, kernel configuration, kernel version and list of installed packages -> then we should make automatic check for example if user uses GPU_DRIVER 1.0 -> assuming that GPU_DRIVER 1.0 is confirmed not working -> output error and ask end-user to update GPU driver, etc..

**Comment by ImperatorS79** [comment]
@Kreyren you can propose to add an issue template for the scripts repo.

**Comment by Kreyren** [comment]
@ImperatorS79 Working on https://github.com/PhoenicisOrg/scripts/pull/815 atm expected push in saturday. 

This can help in theory: https://github.com/RXT067/Scripts/blob/master/KUWDS/KUWDS.sh it's script that is expected to run instead of `wine` to debug wineapp and output results in markdown document.

**Comment by ImperatorS79** [comment]
I meant a file like [https://github.com/PhoenicisOrg/phoenicis/blob/master/.github/issue_template.md](https://github.com/PhoenicisOrg/phoenicis/blob/master/.github/issue_template.md), but for scripts repo. This is what is displayed when you open a new issue.

**Comment by plata** [comment]
@ImperatorS79 yes, we should do that. It is also possible to fill out some information automatically, see https://eric.blog/2016/01/08/prefilling-github-issues/.
